### ISSUE TITLE
Error when clicking "Import Numpy Mocap"

### ISSUE THREAD

**Issue by MaxGodTier** [vltmedia/QuickMocap-BlenderAddon#2]
I seem able to import the plugin now, but when I click "Import Numpy Mocap" it throws this error:
```
Python: Traceback (most recent call last):
  File "C:\Users\Luca\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\QuickMocap\operators\quickmocap_Op_ImportNumpyMocap.py", line 18, in execute
    SMPL_Importer_ = SMPL_Importer(context)
  File "C:\Users\Luca\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\QuickMocap\helpers\smpl_import.py", line 53, in __init__
    addon_prefs = preferences.addons['QuickClean-MocapAddon'].preferences
KeyError: 'bpy_prop_collection[key]: key "QuickClean-MocapAddon" not found'
```

**Comment by vltmedia** [comment]
@MaxGodTier Just put out a new release: [Click Here](https://github.com/vltmedia/QuickMocap-BlenderAddon/releases/tag/0.3.3.1) 

This imports the files and unblocks the pipeline.

Let me know if it works and I can close this issue.

**Comment by MaxGodTier** [comment]
Everything seems to be working now, thank you so much! By the way, I noticed that I can import from QuickMocap only if there's another object in the scene (ie. a cube) to be replaced with, if there're zero objects in the scene then the options will disappear, your YouTube video behaves the same, it would be useful to note that in the README as some people are likely to delete the default cube and get mad when nothing seems to be working.

**Comment by vltmedia** [comment]
@MaxGodTier  Nice! 
We quickly patched the problem you stated: [QuickMocap 0.3.3.2](https://github.com/vltmedia/QuickMocap-BlenderAddon/releases/tag/0.3.3.2).

Closing this issue. Please open up a new issue if anything comes up!
